scripts/build.sh

```diff
@@ -19,6 +22,8 @@ fi
 # Build number from the short hash
 if [ -n "$NOW_GITHUB_COMMIT_SHA" ]; then
   build=$(echo "$NOW_GITHUB_COMMIT_SHA" | cut -c1-7)
+elif [ -n "$GITHUB_SHA" ]; then
```

Comment:
what are those ones for?

Reply:
For some reason... trying to use `git` inside an Github Action was causing the build to fail: https://github.com/aragon/court-dashboard/runs/611481468.

So I just used the relevant environment variables available in an Action.